apps/accounts/forms.py

```diff
@@ -1,9 +1,9 @@
 from django import forms
 from django.contrib.auth import password_validation
 from django.contrib.auth.forms import (  # yapf: disable
-    AuthenticationForm as BaseAuthenticatonForm, PasswordResetForm as BasePasswordResetForm,
-    SetPasswordForm as BaseSetPasswordForm, UserChangeForm as BaseUserChangeForm,
-    UserCreationForm as BaseUserCreationForm
+    AuthenticationForm as BaseAuthenticatonForm, PasswordChangeForm as BasePasswordChangeForm,
+    PasswordResetForm as BasePasswordResetForm, SetPasswordForm as BaseSetPasswordForm,
+    UserChangeForm as BaseUserChangeForm, UserCreationForm as BaseUserCreationForm
```

Comment:
Never realised I could do multiple import as if you group them in a tuple, cool!